core/psa_crypto.c

```diff
@@ -3768,7 +3768,12 @@ psa_status_t psa_verify_hash_abort(
 void mbedtls_psa_interruptible_set_max_ops(uint32_t max_ops)
 {
 
-#if (defined(MBEDTLS_PSA_BUILTIN_ALG_ECDSA) || \
+#if (defined(MBEDTLS_PSA_BUILTIN_ALG_ECDH) || \
+    defined(MBEDTLS_PSA_BUILTIN_KEY_TYPE_ECC_KEY_PAIR_GENERATE) || \
+    defined(MBEDTLS_PSA_BUILTIN_KEY_TYPE_ECC_KEY_PAIR_IMPORT) || \
+    defined(MBEDTLS_PSA_BUILTIN_KEY_TYPE_ECC_KEY_PAIR_EXPORT) || \
+    defined(MBEDTLS_PSA_BUILTIN_KEY_TYPE_ECC_PUBLIC_KEY) || \
```

Comment:
A couple of things on this:
1. please update al the comment on the `#endif` line;
2. ideally I agree with the dependency on `MBEDTLS_PSA_BUILTIN_KEY_TYPE_ECC_KEY_PAIR_xxx`, but it wasn't there before and, according to similar scenarios in this file, `_KEY_PAIR_xxx` is used in conjunction with `_PUBLIC_KEY` when there is no algorithm to be also checked. Perhaps we can skip this check? Uniformize it across the entire file is out of scope for this PR.